grass7/imagery/i.sentinel/i.sentinel.download/i.sentinel.download.py

```diff
@@ -173,8 +173,12 @@ def get_aoi(vector=None):
     if '+proj' not in kv:
         gs.fatal('Unable to get AOI: unprojected location not supported')
     geom_dict = gs.parse_command('v.out.ascii', format='wkt', **args)
```

Comment:
Looks good. One suggestions for a check and warning message,
in case users provide a unsuitable vector map (with more than one or no areas)...

```
topology = gs.parse_command('v.info', map=vector, flags='t')
if int(topology['areas']) < 1:
    gs.fatal(_("No areas found in AOI map <{}>...").format(vector))
elif  int(topology['areas']) > 1:
    gs.warning(_("More than one areas found in AOI map <{}>. \
Using only the first...").format(vector))
```

Reply:
Thanks, PR updated accordingly.

I just wonder if a limit is needed in the v.in.ascii call to really only take the first area?